dll/win32/browseui/internettoolbar.h

```diff
@@ -92,6 +92,7 @@ class CInternetToolbar :
     BOOL                                    fSizing;
     POINT                                   fStartPosition;
     LONG                                    fStartHeight;
+    BrowseUISettings                        *pSettings;
```

Comment:
Where is this initialized?

Reply:
`pSettings` is initialized using a `BWM_SETTINGCHANGE` window message. The lparam contains a pointer to the actual settings struct in its `shellbrowser` parent.

Reply:
Doesn't it need to be initialized to NULL initially in the constructor?

Reply:
Fixed.